installer/win/Brackets.wxs

```diff
@@ -96,7 +104,19 @@
         <!--Setup Folder Structure -->
         <Directory Id='TARGETDIR' Name='SourceDir'>
             <Directory Id='ProgramFilesFolder'>
-                <Directory Id='INSTALLDIR' Name='!(loc.ShortProductName)'/>
+                <Directory Id='INSTALLDIR' Name='!(loc.ShortProductName)'>
+                  <Component Id="UpdatePath"
+                             Guid="1781A625-8ACB-45E7-A8BA-219D81760B2E">
+                    <CreateFolder />
+                    <Environment Id="PATH"
+                                 Action="set"
```

Comment:
This appends `[INSTALLDIR]\Brackets.exe` to the PATH environment variable making Brackets launchable  from command line.